js/components/Table/Row.js

```diff
@@ -4,29 +4,21 @@ import { map } from 'lodash';
 class Row extends React.Component {
   constructor() {
     super();
-
-    this.createRow = this.createRow.bind(this);
   }
```

Comment:
@katiecrane do we need to bind the `newImage` function?

Reply:
@chloealee oh hmm great question. It's working without it binded (bound?) locally?

Reply:
I believe you only need to bind it if you are passing it as a prop to other components to use.

Reply:
ooohh good to know, thanks @sbsmith86 and @katiecrane !